### PR TITLE
[15.0][IMP] account_financial_risk: forward port fixes

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Forward port of https://github.com/OCA/credit-control/pull/161.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@carlosdauden :)

**Review by carlosdauden — APPROVED** [review]

File: account_financial_risk/models/account_invoice.py [hunk 1]
```diff
@@ -59,27 +60,39 @@ def risk_exception_msg(self):
...
         return exception_msg
 
-    def action_post(self):
+    def _post(self, soft=True):
         if (
             self.env.context.get("bypass_risk", False)
             or self.company_id.allow_overrisk_invoice_validation
         ):
-            return super().action_post()
+            return super()._post(soft)
```
Comment (pedrobaeza) [inline account_financial_risk/models/account_invoice.py]: Please always past keyword arguments as such:

```suggestion
            return super()._post(soft=soft)
```

Comment (MiquelRForgeFlow) [reply account_financial_risk/models/account_invoice.py]: but `_post` only has one parameter, it seems doing this is irrelevant :S

**Review by AaronHForgeFlow — APPROVED** [review]
Code review LGTM

**Pull request merged**